apps/studio/data/database/entity-definitions-query.ts

```diff
@@ -87,15 +89,15 @@ export type EntityDefinitionsError = ExecuteSqlError
 export const useEntityDefinitionsQuery = <
   TData extends EntityDefinitionsData = EntityDefinitionsData,
 >(
-  { limit, projectRef, connectionString }: EntityDefinitionsVariables,
+  { schemas, limit, projectRef, connectionString }: EntityDefinitionsVariables,
   options: UseQueryOptions<ExecuteSqlData, EntityDefinitionsError, TData> = {}
 ) => {
   return useExecuteSqlQuery(
```

Comment:
we might need to add the enabled flag here - this query is getting triggered when i have no schemas selected and it leads to a SQL syntax error when this is triggered. Am presuming we now strictly want schemas to be pass into this hook as props so probably can disable this query if none are given